problemtools/verifyproblem.py

```diff
@@ -528,11 +555,16 @@ def check(self, args):
         elif self._data['license'] == 'unknown':
             self.warning("License is 'unknown'")
 
-        if not self._data['grading']['on_reject'] in ['first_error', 'worst_error', 'grade']:
-            self.error("Invalid value '%s' for on_reject policy" % self._data['grading']['on_reject'])
+        if 'on_reject' in self._data['grading']:
+            self.warning('on_reject in problem.yaml is deprecated, use on_reject in testdata.yaml instead.')
```

Comment:
Why not add this one to the list of other deprecation checks below?  Doesn't make a big difference but it keeps deprecation checks in one place plus removes the inconsistency in the wording of the message.